build_tools/webpack_json.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import json
+import logging
+import os
+import sys
+
+from customize_build import load_plugins_from_file
+
+from kolibri.core.webpack.hooks import WebpackBundleHook
+from kolibri.plugins.registry import initialize
+
+logger = logging.getLogger(__name__)
+logging.StreamHandler(sys.stdout)
+logger.setLevel(logging.INFO)
+
+os.environ.setdefault(
+    "BUILD_TIME_PLUGINS", os.path.realpath(os.path.join(os.path.dirname(__file__), "build_plugins.txt"))
+)
+
+def validate_modules(build_list):
+    valid_module_names = [hook.__module__.replace('.kolibri_plugin', '') for hook in WebpackBundleHook().registered_hooks]
+    for module in build_list:
+        if module not in valid_module_names:
+                logger.info('{} did not initialize.'.format(module))
+        else:
+                logger.info('Successfully initialized {}'.format(module))
+
+def main():
+    logger = logging.getLogger(__name__)
+    logging.StreamHandler(sys.stdout)
+    logger.setLevel(logging.INFO)
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--plugin_file', help='the filepath to which you\'d like to run plugins from', type=str, default=None)
```

Comment:
Sorry I'm not sure if I understand this correctly, and I don't really know about webpack. So my understanding is that we can now enable any plugins we want using `python webpack_json.py --plugin_file/--plugins`. Is this usage only for dev mode? Thank you!

Reply:
Yes, although this command is currently invoked from inside the webpack build process. For now, this still uses the environment variable as the primary means for configuration of the list of build plugins, but these command arguments are here to allow more flexibility in the future.

If you want to test this out you can run `BUILD_TIME_PLUGINS=test.txt yarn run devserver ` where `test.txt` is the path to a file with a list of plugin module paths you want to build.

Reply:
thanks for the info!